### PR TITLE
Temp1

### PR DESCRIPTION
**Pull request by tarmolov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,3 +9,6 @@ npm start
 ```
 
 ### Tests
+
```
Comment (tarmolov) [inline README.md]: new comment

Comment (tarmolov) [inline README.md]: comment 2

File: README.md [hunk 2]
```diff
@@ -9,3 +9,6 @@ npm start
...
+### Temp1
+temp1
```
Comment (tarmolov) [inline README.md]: sdfsdfsdfsf

File: README.md [hunk 3]
```diff
@@ -9,3 +9,6 @@ npm start
 ```
 
 ### Tests
+
+### Temp1
```
Comment (zloylos) [inline README.md]: tmp

**Pull request closed without merging**